bessctl/conf/perftest/pktgen_flows.bess

```diff
@@ -0,0 +1,50 @@
+import scapy.all as scapy
+
+#Check shell variables for values, if not set to defaults.
+num_ports = int($SN_PORTS!'1')
+num_cores = int($SN_CORES!'1')
+
+assert(1 <= num_ports <= 16)
+assert(1 <= num_cores <= 4)
+
+# generate flows by varying IP dst 
+num_flows = int($SN_FLOWS!'1')
+assert(1 <= num_flows <= 10000)
+
+src_ether=str($BESS_SRC_ETHER!'02:1e:67:9f:4d:aa')
+dst_ether=str($BESS_DST_ETHER!'02:1e:67:9f:4d:bb')
+# Random ethernet address
+eth = scapy.Ether(src=src_ether, dst=dst_ether)
+# Increment this value
+src_ip=str($BESS_SRC_IP!'192.168.0.1')
+dst_ip=str($BESS_DST_IP!'10.0.0.1')
+ip = scapy.IP(src=src_ip, dst=dst_ip)
+# HTTP flows, dst must be 80, src can be anything
+src_port = int($BESS_SRC_PORT!'10001')
+tcp = scapy.TCP(sport=src_port, dport=80, seq=12345)
+
+host = 'aaa'
+payload_prefix = 'GET /pub/WWW/TheProject.html HTTP/1.1 \r\nHost: http://www.' 
```

Comment:
Perhaps it doesn't matter for NAT, but `Host` field should not include `http://`.